### ISSUE TITLE
Autocomplete has unlabeled input

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#643]
Every page with a search box reports a violation for an input without a label because of this:

`<input type="text" value="" class="search-q q form-control rounded-left tt-hint" data-autocomplete-enabled="true" data-autocomplete-path="/catalog/suggest" readonly="" autocomplete="off" spellcheck="false" tabindex="-1" style="position: absolute; top: 0px; left: 0px; border-color: transparent; box-shadow: none; opacity: 1; background: none 0% 0% / auto repeat scroll padding-box padding-box rgb(255, 255, 255);" dir="ltr">`

which is not in the tab order but is perceivable by the screen reader in the "form controls" menu. 

Not sure what to propose to correct it; typeahead doesn't seem to be enabled. Label it? Hide it? ??

**Comment by peetucket** [comment]
No longer seems to be present in [bl8 upgrade branch](https://github.com/sul-dlss/earthworks/tree/bl8)
